locale/de/locale.cfg

```diff
@@ -1,10 +1,19 @@
 [item-name]
-wooden-wall= Holz-Wand
+wooden-wall= Holzmauer

```

Comment:
this is "woodcutter" in english, the mod name is "wooden wall", this line won't be accepted